src/quad_expr.jl

```diff
@@ -52,6 +52,8 @@ mutable struct GenericQuadExpr{CoefType,VarType} <: AbstractJuMPScalar
     terms::OrderedDict{UnorderedPair{VarType}, CoefType}
 end
 
+Base.ndims(::GenericQuadExpr) = 0
+
```

Comment:
I also had to add this function.

Reply:
That's expected, you should detect it with MA tests. I didn't know QuadExpr were passing MA tests without it before Julia v1.6